src/int/div.rs

```diff
@@ -0,0 +1,335 @@
+//! [`Int`] division operations.
+
+use core::ops::Div;
+
+use subtle::{Choice, CtOption};
+
+use crate::{CheckedDiv, ConstChoice, ConstCtOption, Int, NonZero, Uint};
+
+impl<const LIMBS: usize> Int<LIMBS> {
+    #[inline]
+    /// Base div_rem operation.
+    /// Given `(a, b)`, computes the quotient and remainder of their absolute values. Furthermore,
```

Comment:
```suggestion
    /// Base div_rem operation.
    ///
    /// Given `(a, b)`, computes the quotient and remainder of their absolute values. Furthermore,
```

This will look better in the rustdoc, as it doesn't include the explanative comment in one-line summaries.

Reply:
Done.